tests/main_test.py

```diff
@@ -296,3 +296,75 @@ def test_run_malformed(
         mock_db_manager.assert_called_once_with()
         assert 'Configuration file {} is malformed: {} not found'.format(
             'config', '\'host\'') in str(err.value)
+
+    @mock.patch('hanadb_exporter.utils.systemd_ready')
+    @mock.patch('hanadb_exporter.main.LOGGER')
+    @mock.patch('hanadb_exporter.main.parse_arguments')
+    @mock.patch('hanadb_exporter.main.parse_config')
+    @mock.patch('hanadb_exporter.main.setup_logging')
+    @mock.patch('hanadb_exporter.main.db_manager.DatabaseManager')
+    @mock.patch('hanadb_exporter.main.prometheus_exporter.SapHanaCollectors')
+    @mock.patch('hanadb_exporter.main.REGISTRY.register')
+    @mock.patch('hanadb_exporter.main.start_http_server')
+    @mock.patch('logging.getLogger')
+    @mock.patch('time.sleep')
+    @mock.patch('hanadb_exporter.main.secrets_manager.get_db_credentials')
+    def test_run_secret_manager(
+            self, mock_secret_manager, mock_sleep, mock_get_logger, mock_start_server, mock_registry,
+            mock_exporters, mock_db_manager, mock_setup_logging,
+            mock_parse_config, mock_parse_arguments, mock_logger, mock_systemd):
+
+        mock_arguments = mock.Mock(config='config', metrics='metrics', daemon=False, version=False)
+        mock_parse_arguments.return_value = mock_arguments
+        mock_secret_manager.return_value = {
+                                                'username': 'db_user',
```

Comment:
This doesn't look a correct indentation to pass the pylint checks

Reply:
addressed